src/amo/pages/AddonReviewList/index.js

```diff
@@ -144,36 +172,6 @@ export class AddonReviewListBase extends React.Component<InternalProps> {
     }
   }
 
-  componentDidMount() {
```

Comment:
We still need the same code to run on mount to show reply buttons on the *first* load.

Reply:
Thanks! It is updated.